src/components/header/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../../pages/_app';
+
+const Header = () => {
+  return (
+    <header className="header">
+      <div className="header-container">
+        <div className="header__left-content">
+          <div className="logo-wrapper">
+            <img src="../assets/images/logo.png" alt="Leapfrog" />
```

Comment:
you can create a new file image.js as assets/image.js where you can import the image file.

```
import Logo from 'import Login1 from './images/logo.png';

export { Logo };
```

and import images in js file such that you can use it as:

```
import { Logo } from '../../assets/images';

<img src={Login1} alt="Leapfrog" />
```

Reply:
🆗